frontend/src/components/subject/createSubjectView/body/UserSearchList.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+    <v-text-field
+        :loading="searchLoading"
+        v-model="search"
+        :label="$t('default.search')"
+        :placeholder="$t('create_subject.search_for_instructors')"
+        prepend-inner-icon="mdi-magnify"
+        clearable
+        single-line
+        hide-details
+        variant="outlined"
+        @click:prepend-inner="onSearchIconClicked"
+        @keydown.enter="onSearchIconClicked"
```

Comment:
Dit is niet heel gebruiksvriendelijk, het zou handig zijn als het filteren gebruikt terwijl de gebruiker aan het typen is. Om een grote hoeveelheid gebruikers aan te kunnen zal hier waarschijnlijk throttling voor gebruikt moeten worden.